lib/admin_page/user/mobile_and_web_layout_design/mobile_design.dart

```diff
@@ -0,0 +1,294 @@
+                      ),
+                    ],
+                  ),
+                  Positioned(
+                    top: height*0.0050,
+                    bottom: height*0.0020,
+                    right: height*0.0010,
+                    left: height*0.0010,
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        Container(
+                          height: height * 0.25,
+                          width: width * 0.45,
+                          decoration: BoxDecoration(
+                            color: Colors.white,
+                            // border: Border.all(
+                            //   width: 0.5,
+                            //   color: Colors.blueAccent,
+                            // ),
+                            borderRadius: BorderRadius.circular(10),
+                            boxShadow: const [
+                              BoxShadow(
+                                offset: Offset(0.5, 0.5),
+                                blurRadius: 1,
+                                color: Colors.grey,
+                              ),
+                              BoxShadow(
+                                offset: Offset(-0.5, -0.5),
+                                blurRadius: 1,
+                                color: Colors.grey,
+                              )
+                            ],
+                          ),
+                          child: Column(
+                            children: [
+                              SizedBox(
+                                height: height * 0.02,
+                              ),
+                              Container(
+                                height: height*0.13,
+                                width:height*0.13,
+                                decoration: BoxDecoration(
+                                  color: Colors.white,
+                                  borderRadius: BorderRadius.circular(height*0.13),
+                                  boxShadow: [
+                                    BoxShadow(
+                                      offset: const Offset(0.5, 0.5),
+                                      blurRadius: 0.5,
+                                      color: Colors.white.withOpacity(0.5),
+                                    )
+                                  ],
+                                ),
+                                child: CircleAvatar(
+                                  backgroundImage:
+                                  NetworkImage(widget.user?.image ?? ''),
+                                ),
+                              ),
+                              SizedBox(
+                                height: height * 0.02,
+                              ),
+                              Text(
+                                "${widget.user?.firstName ?? ''} ${widget.user?.lastName}"
```

Comment:
here you are creating a full name by using the first_name or last_name of the user class.  This is a business logic that should be present inside your entity/model class. You can make a function or variable which would return the full name inside the user class, and use the same here.